### PR TITLE
Removed redundant returns

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

File: app/helpers/harvests_helper.rb [hunk 1]
```diff
@@ -19,20 +19,16 @@ def display_human_quantity(harvest)
       if harvest.unit == 'individual' # just the number
         number_to_human(harvest.quantity, strip_insignificant_zeros: true)
       elsif !harvest.unit.blank? # pluralize anything else
-        return pluralize(number_to_human(harvest.quantity, strip_insignificant_zeros: true), harvest.unit)
+        pluralize(number_to_human(harvest.quantity, strip_insignificant_zeros: true), harvest.unit)
```
Comment (CloCkWeRX) [inline app/helpers/harvests_helper.rb]: I'd be kind of inclined to refactor this method to return early, so theres less nested conditionals
```return unless !harvest.quantity.blank? && harvest.quantity > 0```

File: app/helpers/harvests_helper.rb [hunk 2]
```diff
@@ -19,20 +19,16 @@ def display_human_quantity(harvest)
...
-        return "#{number_to_human(harvest.quantity, strip_insignificant_zeros: true)} #{harvest.unit}"
+        "#{number_to_human(harvest.quantity, strip_insignificant_zeros: true)} #{harvest.unit}"
       end
-    else
-      return nil
     end
   end
 
   def display_weight(harvest)
     if !harvest.weight_quantity.blank? && harvest.weight_quantity > 0
```
Comment (CloCkWeRX) [inline app/helpers/harvests_helper.rb]: Ditto

File: app/models/seed.rb [hunk 3]
```diff
@@ -66,17 +66,17 @@ class Seed < ActiveRecord::Base
 
   def tradable?
     if self.tradable_to == 'nowhere'
```
Comment (CloCkWeRX) [inline app/models/seed.rb]: Can be a one liner instead
```self.tradable_to != 'nowhere'```

**Comment by CloCkWeRX** [comment]
Mmm, those tests are being odd. I can't see how https://travis-ci.org/Growstuff/growstuff/jobs/181238816 is failing because of https://github.com/Growstuff/growstuff/pull/1116/commits/5517139c028ac9d9df9dffd9bb6630e52691107c alone.

**Comment by Br3nda** [comment]
I'm running some git gisect on this

**Pull request merged**